core/ffmpeg/verifyInstall.go

```diff
@@ -0,0 +1,32 @@
+package ffmpeg
+
+import (
+	"errors"
+	"fmt"
+	"os"
+)
+
+//VerifyFFMpegPath verifies that the path exists, is a file, and is executable
+func VerifyFFMpegPath(path string) error {
+	stat, err := os.Stat(path)
+
+	if os.IsNotExist(err) {
+		return errors.New("ffmpeg path does not exist")
+	}
+
+	if err != nil {
+		return fmt.Errorf("error while verifying the ffmpeg path: %s", err.Error())
+	}
+
+	if stat.IsDir() {
+		return errors.New("ffmpeg path can not be a folder")
+	}
+
+	mode := stat.Mode()
+	//source: https://stackoverflow.com/a/60128480
+	if mode&0111 == 0 {
```

Comment:
Nice touch!

Reply:
Only thing I am worried about is whether or not this works on Windows too or if we should add a special handler for Windows 🧐